tests/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -22,37 +23,29 @@ local utils = import 'templates/utils.libsonnet';
     git clone https://github.com/huggingface/transformers.git
     cd transformers && pip install .
     git log -1
-    pip install datasets evaluate scikit-learn
-    python examples/pytorch/xla_spawn.py \
-      --num_cores 8 \
-      examples/pytorch/text-classification/run_glue.py \
-      --logging_dir=./tensorboard-metrics \
-      --task_name MNLI \
-      --cache_dir ./cache_dir \
-      --do_train \
-      --do_eval \
-      --num_train_epochs 3 \
-      --max_seq_length 128 \
-      --learning_rate 3e-5 \
-      --output_dir MNLI \
-      --overwrite_output_dir \
-      --logging_steps 30 \
-      --save_steps 3000 \
-      --overwrite_cache \
-      --tpu_metrics_debug \
+    pip install datasets evaluate sklearn
+    pip install -r examples/pytorch/_tests_requirements.txt
   |||,
   local command_copy_metrics = |||
     gsutil -m cp -r ./tensorboard-metrics/* $(MODEL_DIR)
   |||,
   local bert_base_cased = common.Convergence {
     modelName: 'hf-glue-bert-b-c',
+    paramsOverride+:: {
+      model_name_or_path: 'bert-base-cased',
+      per_device_train_batch_size: 64,
+      per_device_eval_batch_size: 64,
+    },
     command: utils.scriptCommand(
```

Comment:
Every test has the same command, right?

```
command: utils.scriptCommand(
    |||
        %s 
        %s 
        %s 
      ||| % [
        command_common,
        utils.toCommandString(self.paramsOverride.trainCommand),
        command_copy_metrics,
      ]
    ),
```

Can you move this into `hf_glue`?

Reply:
Yes, moved